src/states/north_carolina/main.py

```diff
@@ -20,7 +20,7 @@ def parse_county(county):
     'official': county['DirectorName'].title(),
     'emails': [county['Email']],
     'faxes': to_list(county['FaxNum']),
-    'phones': [join_fields(county, ' x', ['OfficePhoneNum', 'OfficePhoneNumExt'])],
+    'phones': to_list(county['OfficePhoneNum']),  # exclude OfficePhoneNumExt
```

Comment:
Can we keep the extensions as a string.  So E.164 format + arbitrary string for phone numbers?